docs/installation.rst

```diff
@@ -70,7 +70,9 @@ variables. ::
 update
 ++++++
 For updating an existing MSS installation to the current version, it is best to install
-it into a new environment.
+it into a new environment. If your current version is not far behind the new version
```

Comment:
strong disagree. This might fail and leave you without a usable installation. It's IMO fine to do it, but we should not suggest it to arbitrary users.

Reply:
I think we should do. I updated the description, may be that is better. I use it since months.